.devops/pipelines/release-northstar.yml

```diff
@@ -0,0 +1,196 @@
+pr: none
```

Comment:
For the moves I’d probably prefer just moving the files in this PR and updating the pipelines that reference them as soon as the PR merges. The release pipeline file in 7.0 (and possibly 6.0 and 5.0) will need to be moved as well since the same ADO pipeline is used for all of them.

Reply:
> The release pipeline file in 7.0 (and possibly 6.0 and 5.0)

This is exactly why i would like to dupe these pipelines and gradually migrate the pipelines one at a time, and delete the current yml templates as pipelines are migrated -> Less chance of blowing everything up at once and it's clearer as I'm deleting pipelines what's actually being migrated